pages/pools/find.tsx

```diff
@@ -0,0 +1,39 @@
+import { NextPage } from 'next';
+import { values } from 'ramda';
+import { FormProvider, useForm } from 'react-hook-form';
+
+import { SEO } from '@/components';
+import { COINS } from '@/constants/coins';
+import { ModalProvider } from '@/context/modal';
+import { useNetwork } from '@/context/network';
+import FindPool from '@/views/find-pool';
+import { IPoolForm } from '@/views/find-pool/find-pool.types';
+
+const FindPoolPage: NextPage = () => {
+  const { network } = useNetwork();
+  const form = useForm<IPoolForm>({
+    mode: 'onBlur',
+    reValidateMode: 'onBlur',
+    defaultValues: {
+      tokenA: {
+        value: '0',
+        ...values(COINS[network])[0],
+      },
+      tokenB: {
+        value: '0',
+        ...values(COINS[network])[1],
+      },
+    },
+  });
+
+  return (
+    <FormProvider {...form}>
+      <ModalProvider>
+        <SEO pageTitle="Find Pool" />
```

Comment:
The page title has changed on Figma to Create Pool

Reply:
We change to Find Pool... Figma not updated